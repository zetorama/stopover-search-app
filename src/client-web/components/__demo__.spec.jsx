import React from 'react'
import { shallow } from 'enzyme'
import Comp from './__demo__'

describe('__demo__ comp', () => {
  test('renders all components', () => {
    const res = shallow(<Comp />)
    expect(res).toMatchSnapshot()
  })
})
