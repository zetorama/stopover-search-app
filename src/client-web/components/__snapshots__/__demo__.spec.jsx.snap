// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`__demo__ comp renders all components 1`] = `
<PageLayout>
  <PageHeader />
  <hr />
  <SearchForm />
  <hr />
  <Welcome />
  <hr />
  <Spiner />
  <hr />
  <SearchResult />
  <hr />
  <SearchResult
    offers={
      Array [
        Object {
          "address": Object {
            "cityName": "PARIS",
            "countryCode": "FR",
            "lines": Array [
              "4 ALLEE DU VERGER",
            ],
            "postalCode": "95700",
          },
          "contact": Object {
            "fax": "33-1-30182201",
            "phone": "33-1-30182200",
          },
          "id": "XKCDG0E0",
          "isAvailable": true,
          "name": "HOLIDAY INN PARIS CDG AIRPORT",
          "offers": Array [
            Object {
              "currency": "USD",
              "id": "8060D9C4598956A01752F8B0C4CC4F2A80ECBA000D1A6DA49AA86918D36D415A",
              "roomType": Object {
                "bedType": "QUEEN",
                "beds": 2,
                "category": "STANDARD_ROOM",
              },
              "totalPrice": "79.99",
            },
          ],
        },
        Object {
          "address": Object {
            "cityName": "ROISSY CDG CEDEX",
            "countryCode": "FR",
            "lines": Array [
              "ROISSYPOLE OUEST",
              "ROUTE DE LA COMMUNE BP 20248",
            ],
            "postalCode": "95713",
          },
          "contact": Object {
            "email": "h0577@accor.com",
            "fax": "33/1/49192900",
            "phone": "33/1/49192929",
          },
          "id": "RTCDGAIR",
          "isAvailable": true,
          "name": "MERCURE PARIS CDG AIRPORT",
          "offers": Array [
            Object {
              "currency": "EUR",
              "id": "D5DE80B70D60BE00543A6AB76640663B1828EFE4660CE2F88D7416EFD63C3E14",
              "roomType": Object {
                "bedType": "DOUBLE",
                "beds": 1,
                "category": "STANDARD_ROOM",
              },
            },
          ],
        },
      ]
    }
  />
  <hr />
  <Error
    errors={
      Array [
        Object {
          "code": "007",
          "title": "Mission aborted, James",
        },
      ]
    }
  />
  <hr />
  <Note
    ctx="success"
  >
    That's all folks!
  </Note>
</PageLayout>
`;
